### PR TITLE
Add sharee list view js tests

### PR DESCRIPTION
**Pull request by tomneedham**
Added some tests to look at the sharee list view

@rullzer @PVince81

### CODE REVIEW

File: core/js/tests/specs/sharedialogshareelistview.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+  var listView;
+  var setPermissionsStub;
+
+	beforeEach(function() {
+		/* jshint camelcase:false */
+		oldAppConfig = _.extend({}, oc_appconfig.core);
+		oc_appconfig.core.enforcePasswordForPublicLink = false;
+
+    $('#testArea').append('<input id="mailPublicNotificationEnabled" name="mailPublicNotificationEnabled" type="hidden" value="yes">');
```
Comment (PVince81) [inline core/js/tests/specs/sharedialogshareelistview.js]: tabs vs spaces ? looks weird here :smile:

**Comment by PVince81** [comment]
The tests look good, please adjust the indents

**Comment by tomneedham** [comment]
:sob: Atom shows it fine... upload to github and it's wrong

**Comment by MorrisJobke** [comment]
> Atom shows it fine... upload to github and it's wrong

Use a proper IDE :see_no_evil: :hear_no_evil: :speak_no_evil:

**Comment by PVince81** [comment]
Even vim gets it right **hides**

**Comment by MorrisJobke** [comment]
I fixed the indentation and rebased on master (to have a working travis test in here).

Lets review this one:)

**Comment by MorrisJobke** [comment]
Tests look good to me :+1:

**Comment by MorrisJobke** [comment]
```
$ bash tests/travis/test_for_app.sh $APP
Test config is not relevant for this change. terminating
The command "bash tests/travis/test_for_app.sh $APP" failed and exited with 1 during .
```

@DeepDiver1975 I don't get why there is an `exit 1` in the travis if the test suite doesn't apply. I will remove this.

File: tests/travis/test_for_app.sh [hunk 2]
```diff
@@ -13,5 +13,4 @@ if git diff ${TRAVIS_COMMIT_RANGE} | grep -- "^+++ b/apps/$APP/"; then
 	echo "Executing this test config ...."
 else
 	echo "Test config is not relevant for this change. terminating"
-	exit 1
```
Comment (MorrisJobke) [inline tests/travis/test_for_app.sh]: cc @DeepDiver1975

Comment (MorrisJobke) [inline tests/travis/test_for_app.sh]: Got it. Otherwise travis just runs fine. Let me check if I can somehow tell travis to quit early.

Comment (MorrisJobke) [inline tests/travis/test_for_app.sh]: Lets handle this in #20121

**Comment by MorrisJobke** [comment]
> @DeepDiver1975 I don't get why there is an exit 1 in the travis if the test suite doesn't apply. I will remove this.

Let's handle this in 20121

**Comment by PVince81** [comment]
:+1:

**Pull request merged**